README.md

```diff
@@ -4,9 +4,13 @@
 
 [![GitHub Release](https://img.shields.io/github/release/ubc-mds/visumorph.svg?style=flat)]()
 [![GitHub Activity](https://img.shields.io/github/last-commit/ubc-mds/visumorph/main.svg?style=flat)]()
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 [![Documentation Status](https://readthedocs.org/projects/visumorph/badge/?version=latest)](https://visumorph.readthedocs.io/en/latest/?badge=latest)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![codecov](https://codecov.io/github/UBC-MDS/VisuMorph/graph/badge.svg?token=rnxgkcvwda)](https://codecov.io/github/UBC-MDS/VisuMorph)
+.. image:: https://www.repostatus.org/badges/latest/active.svg
+   :alt: Project Status: Active – The project has reached a stable, usable state and is being actively developed.
+   :target: https://www.repostatus.org/#active
```

Comment:
I think you added these three lines by mistake?

Reply:
Oh, you are right. Added them, and adapted them, but forgot to delete them. Allow me

Reply:
no worries!

Reply:
Resolved in [9c49585](https://github.com/UBC-MDS/VisuMorph/pull/99/commits/9c495850f801a983e5a0576b35233f20a32cc839).